### PR TITLE
docs: Fix docs about validation enums

### PR DESCRIPTION
**Pull request by KasimAhmic**
Two small fixes:

 - VALIDATION_ACTIONS and VALIDATION_LEVEL weren't being imported in the example
 - VALIDATION_ACTIONS was written as VALIDATION_ACTION

### CODE REVIEW

File: docs/api/schema.md [hunk 1]
```diff
@@ -6,15 +6,7 @@ A schema is what defines a Model and the validation performed on the collection
...
-While the default `_id` property is added with an `ObjectId` type, its type can be customized into a `string` or a `number`
+Creates a schema for a model and define validation options for it.

Read more about the validation [options](https://docs.mongodb.com/manual/core/schema-validation/index.html#behavior)
in the MongoDB docs.

Under the hood it is just a wrapper around the [`object`](api/types.md#object) type with some default properties
(e.g. `_id` and timestamps properties).

While the default `_id` property is added with an `ObjectId` type, its type can be customized into a `string` or a `number`
```
Comment (avaly) [inline docs/api/schema.md:9]: Do you mind fixing these newlines?

**Review by avaly — CHANGES_REQUESTED** [review]

File: docs/api/schema.md (hunk 1 above)
Comment (KasimAhmic) [reply docs/api/schema.md]: Done. Not sure why, but I guess IntelliJ wants to do it's own thing with line endings lol VS Code handled it better it seems.

File: docs/api/schema.md [hunk 2]
```diff
@@ -18,23 +18,23 @@ While the default `_id` property is added with an `ObjectId` type, its type can
...
 
-| Name | Type | Attribute |
-| --- | --- | --- |
-| `properties` | `Record<string, unknown>` | required |
-| `options` | `SchemaOptions` | optional |
-| `options.defaults` | `Partial<TProperties>` | optional |
-| `options.timestamps` | `boolean` | optional |
-| `options.validationAction` | `VALIDATION_ACTION` | optional |
-| `options.validationLevel` | `VALIDATION_LEVEL` | optional |
+| Name                       | Type                      | Attribute |
```
Comment (avaly) [inline docs/api/schema.md:21]: These edits look like they were manual. These files are generated automatically when running `yarn docs`. Please run that instead and commit the resulting files.

**Review by avaly — CHANGES_REQUESTED** [review]

File: docs/api/schema.md (hunk 2 above)
Comment (KasimAhmic) [reply docs/api/schema.md]: So I just did that and now the bad line endings are back :/

**Comment by KasimAhmic** [comment]
Apologies it took so long, but I've re-generated the docs on a Linux machine and there are no new line characters anymore.

**Review by avaly — APPROVED** [review]

**Pull request merged**